keyboards/acheron/themis/readme.md

```diff
@@ -0,0 +1,23 @@
+# Acheron Themis QMK firmware
+
+![athena](https://acheronproject.com/images/athena/IMG_9653.jpg)
```

Comment:
image doesn't render 🙁 - please re-host on Imgur.

Reply:
Fixed in 2ff0be2